.github/workflows/ci.yml

```diff
@@ -101,8 +101,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - mw: 'REL1_39'
-            php: '8.2'
+          - mw: 'REL1_43'
+            php: '8.3'
```

Comment:
To support MW 1.39 we need to use the older class aliases. PHPStan apparently cannot figure them out on its own and [requires additional config](https://phpstan.org/user-guide/discovering-symbols#class-aliases).

Quickest solution for now is to run PHPStan CI on 1.43 which matches the baseline's ignored errors.